README.md

```diff
@@ -70,6 +74,20 @@ You may first need to generate a CircleCI API Token from the [Personal API Token
 ```
 $ circleci setup
 ```
+You should be prompted to enter the _CircleCI API Token_ you generated from Personal API Token tab](https://circleci.com/account/api)
+
+`✔ CircleCI API Token: `
+
+=> `API token has been set.`
+
+`✔ CircleCI Host: https://circleci.com`
+
+=> 
+```
+        CircleCI host has been set.
+        Setup complete. Your configuration has been saved.
```

Comment:
Could you move the output to a single code block section and remove the indention here?